USERS.md

```diff
@@ -0,0 +1,27 @@
+## User Roles in MIQA
+MIQA users have a global role as well as any number of per-project roles.
+
+### Project Roles
+Each Project has a list of members specifying who may view and edit that project and the objects within.
+
+- Users added as "Collaborators" have read-only access. Collaborators cannot edit Experiment notes, upload scans, or submit decisions on scans.
+- Users added as "Members" have write access. Members can edit Experiment notes, upload scans, and submit decisions on scans. There are two classes of Members; A Member can either be "Tier 1 Reviewer" or "Tier 2 Reviewer".  Tier 1 Reviewers can submit a decision that a scan is either "usable" or "questionable". Tier 2 Reviewers can submit a decision that a scan is "usable", "unusable", or "usable-extra".
```

Comment:
Currently, no one can "upload" scans. I guess this is a description of what we are going to add soon?

Reply:
Oh yeah, good point. branch is in progress. I'll take this out for now.